### PR TITLE
Improve release notes

### PR DESCRIPTION
**Pull request by NickDraper**
**Description of work.**

Formatting updates to the release notes.

No Code changes, just rst in the release notes.

**Report to:**nobody

**To test:**

check the rleease notes look ok, especially where there are changes

No issue

This does not require release notes that would be a circular argument

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

File: docs/source/release/v4.0.0/diffraction.rst [hunk 1]
```diff
@@ -5,31 +5,24 @@ Diffraction Changes
 .. contents:: Table of Contents
    :local:
 
+Single Crystal Diffraction
```
Comment (martyngigg) [inline docs/source/release/v4.0.0/diffraction.rst]: The SCD section has now got 2 `Improvements` sections that can be merged I think?

**Review by martyngigg — CHANGES_REQUESTED** [review]
Just a minor section merge issue

**Review by martyngigg — APPROVED** [review]
Looks good :+1:

**Comment by NickDraper** [comment]
@peterfpeterson Could you gatekeeper this?

**Pull request merged**